fastserde/avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastSerdeCache.java

```diff
@@ -405,12 +445,13 @@ public Object deserialize(Object reuse, Decoder d) throws IOException {
    *
    * @param writerSchema writer schema
    * @param readerSchema reader schema
+   * @param modelData Provides additional information not available in the schema, e.g. conversion classes
    * @return a fast deserializer
    */
-  public FastDeserializer<?> buildFastGenericDeserializer(Schema writerSchema, Schema readerSchema) {
```

Comment:
Since the method is public, why not leave the original method signature and introduce the extended method? We might break some existing non-standard usages of this class.

Reply:
Changed in commit `[review-fixes] Part 2`